data/859/018/65/85901865.geojson

```diff
@@ -4,6 +4,8 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:deprecated":"12-21-2020",
```

Comment:
This looks like a legitimate neighbourhood. Are we sure we want to deprecate / mark as not current?

Reply:
This (https://spelunker.whosonfirst.org/id/85901865/) is actually an archaically spelled street (now it is spelled Popesti-Romani) from a small town near Bucharest which is called - Popesti-Leordeni (https://www.google.com/maps/place/Strada+Pope%C5%9Fti-Romani,+Pope%C8%99ti-Leordeni+077160/@44.3813966,26.1676669,16.3z/data=!4m5!3m4!1s0x40b1fddaca55d319:0x9aafdfd48fee12e3!8m2!3d44.3841284!4d26.1651532)
There is no such neighbourhood in Bucharest.